### ISSUE TITLE
HTML5 + Connection loss = successfully uploaded

### ISSUE THREAD

**Issue by joansol** [moxiecode/plupload#499]
If the connection get loss while uploading, the HTML5 uploader (in Chrome) checks the upload as successfull; same situation using Flash uploader (IE9) reports an error while uploading.

**Comment by jayarjo** [comment]
Has been adresed by: https://github.com/moxiecode/plupload/commit/5538f383d7c20cb3893a48cd8d765120cef9f977
